xwiki-pro-macros-confluence-bridges/xwiki-pro-macros-confluence-bridges-ui/src/main/resources/Confluence/Macros/HtmlComment.xml

```diff
@@ -0,0 +1,405 @@
+<?xml version="1.1" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.5" reference="Confluence.Macros.HtmlComment" locale="">
```

Comment:
Did you export the page using a 14.10 xwiki instance?

Reply:
Now I developed with a 16.04 instance. Should I re-export from a 14.10 instance ?

Reply:
Yes. The current parent of the pro-macros is 13.10. However, we need to upgrade it to 14.10 asap. It should be ok for you export this macro from a 14.10 instance

Reply:
Did a import/export from a 14.10 instance. I still have the `xwikidoc version="1.5"` but some class fields was updated.